src/Engine/Manager/SoundManager.cpp

```diff
@@ -65,28 +68,30 @@ void SoundManager::CheckError(PaError err) {
 }
 
 void SoundManager::Update(float deltaTime) {
-    
+
     // Number of samples to process dependant on deltaTime
     int numSamples = int(SAMPLE_RATE * deltaTime);
 
-    
+
     // Update sound sources.
     for (Component::SoundSource* sound : soundSources.GetAll()) {
-        
+
         if (sound->shouldPlay) {
 
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') comma should be followed by whitespace [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT009)